package.json

```diff
@@ -16,31 +16,28 @@
     "curl",
     "debug"
   ],
-  "author": "delirius325",
+  "author": "Abhishek Rajeshirke <https://github.com/colidarity>",
   "contributors": [
     {
-      "name": "Justin \"jayceekay\"",
-      "url": "https://github.com/jayceekay"
-    },
-    {
-      "name": "Peter \"binggg\" Zhao",
-      "url": "https://github.com/binggg"
+      "name": "Abhishek Rajeshirke",
+      "url": "https://github.com/colidarity"
     }
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/delirius325/axios-curlirize/issues"
   },
   "homepage": "https://github.com/delirius325/axios-curlirize#readme",
-  "dependencies": {},
-  "devDependencies": {
+  "dependencies": {
     "axios": ">=0.19.0",
-    "expect": "^23.1.0",
-    "express": "^4.16.3",
-    "mocha": "^5.2.0",
+    "express": "^4.16.3"
```

Comment:
`express` shouldn't be a direct dependency. If anything it should be a devDependency.